internal/resources/metal/project_ssh_key/resource.go

```diff
@@ -123,7 +122,7 @@ func (r *Resource) Update(
 	resp *resource.UpdateResponse,
 ) {
 	r.Meta.AddFwModuleToMetalUserAgent(ctx, req.ProviderMeta)
```

Comment:
All these `AddFwModuleToMetalUserAgent` also need to be changed to add the module name to the equinix-sdk-go User-Agent instead